lib/fuelsdk/objects.rb

```diff
@@ -36,21 +36,21 @@ def perform options
     module Rest
       module Read
         def get
-          client.rest_get id, Array.wrap(properties)
+          client.rest_get id, properties
         end
       end
 
       module CUD
         def post
-          client.rest_post id, Array.wrap(properties)
+          client.rest_post id, properties
         end
 
         def patch
-          client.rest_patch id, Array.wrap(properties)
+          client.rest_patch id, properties
         end
 
         def delete
-          client.rest_delete id, Array.wrap(properties)
```

Comment:
curious the issues you were seeing with `Array.wrap()`. I vaguely remember something about this causing issues in SFApi last time I dug into this gem